backend/dataall/aws/handlers/quicksight.py

```diff
@@ -124,13 +124,41 @@ def create_quicksight_default_group(AwsAccountId):
         return group
 
     @staticmethod
-    def describe_group(client, AwsAccountId):
+    def create_quicksight_group(AwsAccountId, GroupName):
```

Comment:
can we replace calls to `create_quicksight_default_group(AwsAccountId)` with your new function `create_quicksight_group(AwsAccountId,"dataall")` to avoid redundancy in the code ?

Reply:
Good idea, done!